docs/Community/conduct/ConflictPolicy.md

```diff
@@ -0,0 +1,26 @@
+Conflict Resolution
+===================
+
+We do not believe that all conflict is bad; healthy debate and disagreement often
+yield positive results. However, it is never okay to be disrespectful or to engage
+in behavior that violates the project’s [Code of Conduct](CodeOfConduct.md).
+
+If you see someone violating the code of conduct, you are encouraged to address
+the behavior directly with those involved. Many issues can be resolved quickly and
+easily, and this gives people more control over the outcome of their dispute. If
+you are unable to resolve the matter for any reason, or if the behavior is
+threatening or harassing, report it. We are dedicated to providing an environment
+where participants feel welcome and safe.
+
+Reports can be made to any member of the [Community Governance Committee](
```

Comment:
In your Community/CodeOfConduct.md lines 8-11 say: "Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to the Open Enclave [Community Governance Committee](governance/README.md) or to the [Confidential Computing Consortium](https://github.com/confidential-computing/governance/blob/master/code-of-conduct.md)."

But here only mentions the former.  We should be consistent, whichever we use.

Reply:
I disagree that this needs to be so formal and consistent. It's community documentation, not a C header file.